src/sharedm/headers1/headers1.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-headers1',
+  templateUrl: './headers1.component.html',
+  styleUrls: ['./headers1.component.css']
+})
+export class Headers1Component implements OnInit {
```

Comment:
Make component name meaningful `Header1` doesn't make sense for the code can we have `SideNav`